lib/matrix.gi

```diff
@@ -3651,8 +3651,13 @@ end );
 #F  RandomUnimodularMat( [rs ,] <m> )  . . . . . . . random unimodular matrix
 ##
 InstallGlobalFunction( RandomUnimodularMat, function ( arg )
-    local  rs, m, mat, c, i, j, k, l, a, b, v, w, gcd;
+local  rs, m, mat, c, i, j, k, l, a, b, v, w, gcd,dom;
```

Comment:
[Minor] Sorry for the nitpicking, but why there are spaces between all commas except the last one?